src/plone/restapi/serializer/utils.py

```diff
@@ -38,3 +41,10 @@ def uid_to_url(path):
         if a_href:
             return a_href
     return href
+
+
+def get_portal_type(portal_type, request=None):
```

Comment:
I'd suggest calling this `get_portal_type_title`, since `portal_type` is often used to refer to the id of the FTI, and this is not that.